src/pages/assessment/CreateAssessmentModal.tsx

```
@@ -0,0 +1,307 @@
+    // Reset form when closing
+    setAssessmentName('');
+    setSelectedFile(null);
+    setFilename('');
+    setNameError('');
+    setFileError('');
+    setSelectedEnvironmentId('');
+    onClose();
+  };
+
+  // Check if form is valid for button state
+  const isFormValid = assessmentName.trim() && 
```

ESLint

[error] Replace `·assessmentName.trim()·&&·` with `⏎····assessmentName.trim()·&&`

(prettier/prettier)